adagrams/game.py

```diff
@@ -1,11 +1,84 @@
+
+import random
 def draw_letters():
-    pass
+    distribution_of_letters = {"A" : 9, "N" : 6, "B" : 2, "O" : 8, "C" : 2, "P" : 2, "D" : 4 ,"Q" : 1, "E" : 12, "R" : 6, "F" : 2, "S" : 4, "G" : 3, "T" : 6, "H" : 2, "U" : 4, "I" : 9, "V" : 2, "J" : 1 , "W" : 2, "K" : 1, "X" : 1, "L" : 4, "Y" : 2, "M" : 2, "Z" : 1}
```

Comment:
Consider declaring this as a global "constant" (maybe `POOL_OF_LETTERS`) outside this function. 

Doing so wouldn't really save on runtime (since we would then need to modify the counts and make a copy at the start of each call to this method), but it *does* declutter the method a bit by allowing us to move the large data structure to some innocuous corner of our file.